scripts/apitypings/testdata/genericslice/genericslice.ts

```diff
@@ -0,0 +1,10 @@
+// From codersdk/genericslice.go
+export interface Bar {
+  readonly Bar: string
+}
+
+// From codersdk/genericslice.go
+export interface Foo<R extends any> {
```

Comment:
Not for the future: `R extends any` actually doesn't do anything at all. Since `any` is a top type, everything in TypeScript extends `any`, so the constraint provides zero protections